test/suite/test_hs06.py

```diff
@@ -401,7 +401,7 @@ def test_hs_instantiated_modify(self):
             self.assertEqual(cursor[self.create_key(i)], expected)
         self.session.rollback_transaction()
 
-    def test_hs_modify_birthmark_is_base_update(self):
+    def test_hs_modify_stable_is_base_update(self):
```

Comment:
The test doesn't make sense in durable history. However, since it captures this problem, it is still worth to keep it in the suite.